template/web/app/paymaster-bundler/_components/Capabilities.tsx

```diff
@@ -0,0 +1,25 @@
+import { useAccount } from 'wagmi';
+import { useCapabilities } from 'wagmi/experimental';
+
+/**
+ * Capabilities component displays the capabilities of the current account.
+ *
+ * @returns {JSX.Element} The rendered component displaying the capabilities or a message if none are found.
+ */
+export function Capabilities() {
```

Comment:
I am always a bit confused of this part. Why do we need to show `Capabilities`?

Reply:
We definitely do not need to show this. I think it's nice to have access to, especially as a developer. Perhaps we can just leave this as a utility function but avoid defaulting to displaying the data.

Reply:
it feels something all pages can benefit. Maybe a shared component

Reply:
Agreed! I created a new shared component folder called `SmartWallets` and placed `Capabilities` in it. We can use this folder to put any other general Smart Wallet related functionality.